README.md

```diff
@@ -80,12 +80,19 @@ var pageLevelParams = {
         )
 };
 
+// Your codes
```

Comment:
Remove these from readme, as per the comment on receiver.html

Reply:
+